src/collection-view.js

```diff
@@ -279,19 +268,55 @@ const CollectionView = Backbone.View.extend({
       this._showEmptyView();
     } else {
       this.triggerMethod('before:render:children', this);
-      this._startBuffering();
       this._showCollection(models);
-      this._endBuffering();
       this.triggerMethod('render:children', this);
     }
   },
 
+  _createView(model, index) {
+    const ChildView = this._getChildView(model);
+    const childViewOptions = this._getChildViewOptions(model, index);
+    const view = this.buildChildView(model, ChildView, childViewOptions);
+    view._parent = this;
+
+    monitorViewEvents(view);
+
+    // set up the child view event forwarding
+    this._proxyChildEvents(view);
```

Comment:
![noproxy](https://cloud.githubusercontent.com/assets/833429/17505239/c1e4df16-5db4-11e6-8760-4dca4c1d6a24.gif)
without proxy (it fails several test, btw)

with proxy
![withproxy](https://cloud.githubusercontent.com/assets/833429/17505290/22efe878-5db5-11e6-8d22-96fd040300dc.gif)

The difference isn't drastic (about 100ms for 3000 views)